specification/schemas/BaseShipmentItem.json

```diff
@@ -52,6 +52,15 @@
       "example": 135,
       "description": "Weight in grams of a single item within item resource. Should be multiplied by quantity to get total weight."
     },
+    "item_weight_unit": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "mg",
+      "description": "Unit of measurement for item_weight. Options are: mg, g, kg, oz, lb.",
```

Comment:
Because of this example, all shipment example jsons show `mg`, but I think the default `g` is a better example.

Also, we can make this attribute an enum, then the description does not have to mention the possible values.